docs/introduction/how-to-run-abs-code.md

```diff
@@ -111,9 +117,71 @@ echo("hello")
 $
 ```
 
+## ABS Init File
+
+When the ABS interpreter starts running, it will load an optional
+ABS script as its init file. The ABS init file path can be 
+configured via the OS environment variable `ABS_INIT_FILE`. The
+default value is `ABS_INIT_FILE=~/.absrc`.
+
+If the `ABS_INIT_FILE` exists, it will be evaluated before the
+interpreter begins in both interactive REPL or script modes.
+The result of all expressions evaluated in the init file become
+part of the ABS global environment which are available to command
+line expressions or script programs.
+
+Also, note that the `ABS_INTERACTIVE` global environment variable
+is pre-set to `true` or `false` so that the init file can determine
+which mode is running. This is useful if you wish to set the ABS REPL
+command line prompt or history configuration variables in the init file.
+This will preset the prompt and history parameters for the interactive
+REPL. See [REPL Command History](#REPL_Command_History) above.
+
+### Configuring the ABS REPL Command Line Prompt
+The ABS REPL command line prompt may be configured at start up using
+`ABS_PROMPT_LIVE_PREFIX` and `ABS_PROMPT_PREFIX` variables from either
+the ABS or OS environments. The default values are
+`ABS_PROMPT_LIVE_PREFIX=false` and `ABS_PROMPT_PREFIX="⧐  "`.
+
+REPL `live prompt` mode follows the current working directory 
+set by `cd()` when `ABS_PROMPT_LIVE_PREFIX=true` and the
+`ABS_PROMPT_PREFIX` variable contains a `template string`.
+A `template string` may contain the following named placeholders:
+* `{user}`: the current userId
+* `{host}`: the local hostname
+* `{dir}`:  the current working directory following `cd()`
+
+REPL `static prompt` mode will be configured if `ABS_PROMPT_PREFIX`
+contains no `template string` or if `ABS_PROMPT_LIVE_PREFIX = false`.
+The `static prompt` will be the value of the `ABS_PROMPT_PREFIX` string
+(if present) or the default prompt `"⧐  "`. Static prompt mode is
+the default for the REPL.
+
+For example, you can create a `bash`-style live prompt: 
```

Comment:
I see this example is here, can we move it after describing the live-prompt mode?

Reply:
Done